docs/cli/commands.md

```diff
@@ -102,6 +102,10 @@ Slash commands provide meta-level control over the CLI itself.
 - [**`/theme`**](./themes.md)
   - **Description:** Open a dialog that lets you change the visual theme of Gemini CLI.
 
+- **`/install`**
+  - **Description:** Install VS Code themes from marketplace URLs.
+  - **Usage:** `/install <marketplace-url>`
```

Comment:
this needs to be added as sub-command to `/theme` above, see other commands in this file as ref